packages/components/src/input-control/styles/input-control-styles.js

```diff
@@ -42,13 +42,16 @@ const rootLabelPositionStyles = ( { labelPosition } ) => {
 	}
 };
 
+// &&&&&&& added to force application of label positioning styles
 export const Root = styled( Flex )`
-	position: relative;
-	border-radius: 2px;
+	&&&&&&& {
```

Comment:
What's the original rule we want to override here? 7 `&` seems really weird 😅.

I'm thinking maybe we can just inline the exact parent rule here so that it makes more sense in the code.

```css
/* To override the rule in some-places/packages/some-components/style.scss#L87 */
.some-parent .nested-rule & {
  /* ... */
}
```